tester_app/config/initializers/react_on_rails.rb

```diff
@@ -0,0 +1,28 @@
+# Shown below are the defaults for configuration
+ReactOnRails.configure do |config|
+  # Client bundles are configured in application.js
+
+  # Server rendering:
+  # Server bundle is a single file for all server rendering of components.
+  # If you wish to use render_js in your views without any file, set this to "" to avoid warnings.
```

Comment:
@justin808 now that I think about this, isn't this not true unless you're passing `prerender: true`, in which case you *are* server rendering, so whatever?